Lecture4/App.js

```diff
@@ -1,14 +1,41 @@
 import React from "react";
+import { useState } from "react"
 import { createRoot } from "react-dom/client";
```

Comment:
Appreciate it. I have always in my life imported ReactDOM. But you've no use of it here I think.